.github/workflows/test.main.kts

```diff
@@ -67,8 +78,11 @@ workflow(
         runsOn = UbuntuLatest,
     ) {
         uses(action = Checkout())
-        run(name = "Check for all actions") {
-            validateTypings()
+        run(
+            name = "Check for actions",
+            env = mapOf("base_ref" to expr { github.base_ref })
```

Comment:
I like this workaround 😄
Let me include `base_ref` in the next release of the lib, so that we can get rid of it:

```suggestion
            // TODO: replace this workaround once base_ref can be accessed natively:
            // https://github.com/typesafegithub/github-workflows-kt/issues/1946
            env = mapOf("base_ref" to expr { github.base_ref })
```

Reply:
It's not so much a work-around, it is the way to get something transported from the context to the Kotlin logic that is not explicitly supported by the lib. That way you can get all context values into it. :-)